### PR TITLE
create services.

### PR DESCRIPTION
**Pull request by maryammokhtari**

### CODE REVIEW

File: src/main/java/com/digistore/ecommerce/service/mapper/CategoryMapper.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.digistore.ecommerce.service.mapper;
+
+import com.digistore.ecommerce.repository.entity.Category;
+import com.digistore.ecommerce.service.dto.CategoryRequest;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+
+@Mapper(componentModel = "spring")
```
Comment (HamedRahmani) [inline src/main/java/com/digistore/ecommerce/service/mapper/CategoryMapper.java]: please change your mapper to something like this: you can read this [docs](https://www.baeldung.com/mapstruct-ignore-unmapped-properties) for more info.

```
//@MapperConfig(componentModel = MappingConstants.ComponentModel.SPRING)
@Mapper(unmappedTargetPolicy = ReportingPolicy.IGNORE ,componentModel = MappingConstants.ComponentModel.SPRING)
public interface CategoryMapper {
 
    Category categoryRequestToCategory(CategoryRequest categoryRequest);

}
```

File: src/main/java/com/digistore/ecommerce/service/CategoryService.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    }
+    public Category findById(Long id){
+        Optional<Category> category=categoryRepository.findById(id);
+        if(category.isPresent()){
+            return category.get();
+        }
+        throw new ResourceNotFoundException(CATEGORY_IS_NOT_EXIST);
+    }
+    public Category create(CategoryRequest categoryRequest){
+        Category category= categoryMapper.categoryRequestToCategory(categoryRequest);
```
Comment (HamedRahmani) [inline src/main/java/com/digistore/ecommerce/service/CategoryService.java]: please format the codes.

Comment (maryammokhtari) [reply src/main/java/com/digistore/ecommerce/service/CategoryService.java]: Done.

File: src/main/java/com/digistore/ecommerce/service/mapper/CategoryMapper.java (hunk 1 above)
Comment (maryammokhtari) [reply src/main/java/com/digistore/ecommerce/service/mapper/CategoryMapper.java]: Done.

**Review by HamedRahmani — APPROVED** [review]

**Pull request merged**